docs/proposals/reduce-multitenancy-cost.md

```diff
@@ -1,23 +1,25 @@
 ---
-title: "XXXX: Tenancy multiplexing"
+title: "0003: Tenancy multiplexing"
 description: "Tenancy multiplexing"
 draft: true
 ---
 
-# XXXX: Tenancy multiplexing
+# 0003: Tenancy multiplexing
 
 **Authors:** @wilfriedroset @vaxvms @bubu11e
 
 **Date:** 09/2024
 
-**Sponsor(s):** @username of maintainer(s) willing to shepherd this MID
+**Sponsor(s):** @dimitarvdimitrov
```

Comment:
@56quarters i think you were involved in the PR initially: https://github.com/grafana/mimir/pull/9149. Are you ok transitioning this to "accepted"?

Reply:
discussed with @56quarters in a meeting that we are ok with accepting the proposal